modules/python/src2/cv2.cpp

```diff
@@ -1750,6 +1750,152 @@ static PyObject *pycvCreateButton(PyObject*, PyObject *args, PyObject *kw)
 }
 #endif
 
+#ifdef HAVE_OPENCV_DNN
+
+class pycvLayer CV_FINAL : public dnn::Layer
```

Comment:
This code should go here too: https://github.com/opencv/opencv/tree/3.4.1/modules/dnn/misc/python

If some "common" code is required from cv2.cpp then it should be extracted into cv2.hpp file and included properly.

Reply:
@alalek, thanks for review! Replaced it.